<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblHost">
      <property name="text">
       <string>Hostname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtHostname">
      <property name="text">
       <string>ponr.fi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblUser">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtUsername"/>
    </item>
    <item>
     <widget class="QLabel" name="lblPass">
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblDir">
      <property name="text">
       <string>Directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtDirectory"/>
    </item>
    <item>
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string>Status</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
